collect/central_collector.go

```diff
@@ -121,7 +121,7 @@ func (c *CentralCollector) Start() error {
 	c.samplersByDestination = make(map[string]sample.Sampler)
 
 	// The cycles manage a periodic task and also provide some test hooks
-	c.metricsCycle = NewCycle(c.Clock, c.Config.GetSendTickerValue(), c.done)
+	c.metricsCycle = NewCycle(c.Clock, 2*c.Config.GetSendTickerValue(), c.done)
```

Comment:
Whats this multiplication doing?

Reply:
Maybe this is too confusing, I will remove it now that we have a separate config for redis metrics

Reply:
I removed the comment in the PR description too